package.json

```diff
@@ -191,6 +191,7 @@
     "prop-types": "^15.5.8",
     "react-clone-referenced-element": "^1.0.1",
     "react-devtools-core": "^3.4.0",
+    "react-native-local-cli": "^0.57.0",
```

Comment:
Do you expect that the CLI and react-native will continue to be versioned together?

Reply:
I agree with having separate versioning, but this means that the new repo must have a table of compatibility of some sort

Reply:
Don't think it's *needed* at the beginning since React Native contains a dependency on CLI for now. It is similar to the way we handle Metro - it also doesn't have compatibility table on React Native.

However, I do think keeping a `peerDependency` is going to be the right choice to make it easier to sync CLI and React Native versions together.

Reply:
FYI I already published `1.0.0-alpha.0` version and updated this PR.